### DISCUSSION TITLE
API splitted in multiple files

### DISCUSSION THREAD

**Discussion by edelagnier** [spec-first/connexion#1950] [Q&A]
Hi ! 
Thanks for this project, I've set it up a few month ago and it's a pleasure to use.
My question is, my api.yaml was starting to get too big so I decided to separate the endpoints and the schemas. 
```
application/json:
    schema:
      $ref: "model_std.yaml#/Overview"
```
But it returns this error : 
`No such file or directory: '/model_std.yaml'`
I tried to add each file as a api but the model_std.yaml isn't a valid specification by itself. Would you know how I should proceed ?

Thank you and have a nice day

Edit : I tried to divide my api into multiple valid api instead but then the swagger-ui only find the last one (I cant't use different basepath for each because of existing third party services calling my API).

**Comment by RobbeSneyders** [comment]
This is possible since https://github.com/spec-first/connexion/pull/1648.

Please provide a minimal reproducible example if it is not working for you.
